README.md

```diff
@@ -15,10 +15,13 @@ _Add bullet points with the initial features that defines the core of the applic
 
 ## Change Log
 _`Classes`, `structs`, `interfaces` and `enums` etc. are to be linked to the project WiKi Page._
- - **[v0.0.0](LinkToGitHubTag)**
+ - **[v0.1.0-dev](https://github.com/ZhakalenDk/FourWheels/releases/tag/v0.1.0-Dev)** Administration af kundder
```

Comment:
Make a seperator between Version number and the title.

Example:
`**[v0.1.0-dev](https://github.com/ZhakalenDk/FourWheels/releases/tag/v0.1.0-Dev)**: Administration af kundder`

Reply:
Done